## Lixin Cheng (程立新)
Email: easonlcheng@gmail.com

Address: Department of critical care medicine, Shenzhen People's Hospital.

I am a Data Scientist with over 10 years of experience in omics and biotech. Interested in all things related to data.

`RA recruitment:` I am looking for Research Assistants to work together computational biology and bioinformatics. Please email me if interested.

## Research Interests:
* Identifying coding & non-coding biomarkers and signatures for complicated diseases.
* Moonlighting RNAs and proteins.
* Understanding how gene networks are structured in diseases by combining computational algorithms and biological context.
* Quantifying and characterizing gene products in different genetic backgrounds and physiological conditions through subcellular localization, interaction diversity, and expression dynamics in cells.
* Microbiota and metagenome.

## Education:
* Jan 2014 – Jan 2018, PhD (Computational Biology), Department of Computer Science and Engineering, The Chinese University of Hong Kong.
* 2008 – 2011, MPhil (Bioinformatics), College of Bioinformatics Science and Technology, Harbin Medical University.
* 2003 – 2008, BSc (Biotechnology), Department of Biotechnology, Harbin Medical University.

## Position:
* Mar 2019 – present, PI, Department of Critical Care Medicine, Shenzhen People’s Hospital.
* May 2018 – Feb 2019, Postdoctoral Fellow, School of Life Sciences, The Chinese University of Hong Kong.
* May 2012 – Dec 2013, Research Assistant, School of Public Health, Hong Kong University.
* Aug 2011 – Apr 2012, Research Assistant, Department of Electronic and Information Engineering, The Hong Kong Polytechnic University. 

## Selected Publications:
(**#** corresponding author; **IF**, Impact Factor)

* Xueyan Liu, Yong Xu, Ran Wang, Sheng Liu, Jun Wang, YongLun Luo, Kwong-Sak Leung, and **Lixin Cheng#**. A network-based algorithm for the identification of moonlighting non-coding RNAs and its application in sepsis. ***Briefings in Bioinformatics, 2019. (IF=9.1)***

* Jun Wang, Xiuqing Zhang, **Lixin Cheng#**, Yonglun Luo#. An overview and metanalysis of Machine and Deep Learning-Based CRISPR gRNADesign Tools. ***RNA Biology, 2019. (IF=5.5)***

* Jun Wang, Xi Xiang, Lars Bolund, Xiuqing Zhang, **Lixin Cheng#**, Yonglun Luo#. GNL-Scorer: A generalized model for predicting CRISPR on-target activity by machine learning and featurization. ***Journal of Molecular Cell Biology, 2019. (IF=4.7)***

* Qiong Zhao, Jinbo Shen, Caiji Gao, Yong Cui, Yongyi Wang, Jie Cui, **Lixin Cheng**, Wenhan Cao, Ying Zhu, Shuxian Huang, Qianzi Zhou, Cheuk Ka Leong, King Pong Leung, Xuemei Chen & Liwen Jiang#. RST1 is a FREE1 Suppressor that Negatively Regulates Vacuolar Trafficking in Arabidopsis. ***Plant Cell, 2019. (IF=8.2)***

* **Lixin Cheng**, Pengfei Liu, Dong Wang, and Kwong-Sak Leung#. Exploiting locational and topological overlap model to identify modules in human protein interaction network. ***BMC Bioinformatics, 2019. (IF=2.2)***

* **Lixin Cheng#**, Kwong-Sak Leung#. Identification and characterization of moonlighting long non-coding RNAs based on RNA and protein interactome. ***Bioinformatics, 2018. (IF=7.3)***

* **Lixin Cheng#**, Kwong-Sak Leung#. Quantification of non-coding RNA target localization diversity and its application in cancers. ***Journal of Molecular Cell Biology, 2017. (IF=6.0)***

* **Lixin Cheng**, Pengfei Liu, and Kwong-Sak Leung#. SMILE: a novel procedure for subcellular module identification with localization expansion. ***IET System Biology, 2017 (Also in ACM-BCB 2017)***

* **Lixin Cheng**, Kaili Fan, Yan Huang, Dong Wang#, and Kwong-Sak Leung#. Full characterization of localization diversity in human protein interactome. ***Journal of Proteome Research, 2017. (IF=4.3)***

* **Lixin Cheng**, Xuan Wang, Pak-Kan Wong, Kwan-Yeung Lee, Le Li, Bin Xu, Dong Wang#, and Kwong-Sak Leung#.   ICN: A normalization method for gene expression data considering the over-expression of informative genes. ***Molecular BioSystems, 2016. (IF=2.8)***

* **Lixin Cheng**, Leung-Yau Lo, Nelson Tang, Dong Wang#, and Kwong-Sak Leung#.   CrossNorm: a novel normalization strategy for microarray data in cancers. ***Scientific Reports, 2016. (IF=5.6)***

### Conference:
* **Lixin Cheng**, Pengfei Liu, and Kwong-Sak Leung. SMILE: a novel procedure for subcellular module identification with localization expansion. ***ACM-BCB’17, August 20-23, 2017, Boston, MA, USA.***

