## Lixin Cheng (程立新)
Department of critical care medicine, Shenzhen People's Hospital.

Email: easonlcheng@gmail.com

`RA recruitment:` I am looking for research assistants to work together computational biology and bioinformatics. Please email me if interested.

## Position:
* Mar 2019 – present, PI, Department of Critical Care Medicine, Shenzhen People’s Hospital.
* May 2018 – Feb 2019, Postdoctoral Fellow, School of Life Sciences, The Chinese University of Hong Kong.
* May 2012 – Dec 2013, Research Assistant, School of Public Health, Hong Kong University.
* Aug 2011 – Apr 2012, Research Assistant, Department of Electronic and Information Engineering, The Hong Kong Polytechnic University. 

## Education:
* Jan 2014 – Jan 2018, PhD (Computational Biology), Department of Computer Science and Engineering, The Chinese University of Hong Kong.
* 2008 – 2011, MPhil (Bioinformatics), College of Bioinformatics Science and Technology, Harbin Medical University.
* 2003 – 2008, BSc (Biotechnology), Department of Biotechnology, Harbin Medical University.

## Publications: (# corresponding author)
* Jun Wang, Xiuqing Zhang, Lixin Cheng#, Yonglun Luo#. An overview and metanalysis of Machine and Deep Learning-Based CRISPR gRNADesign Tools. RNA Biology, 2019. (IF=5.5)

* Jun Wang, Xi Xiang, Lars Bolund, Xiuqing Zhang, Lixin Cheng#, Yonglun Luo#. GNL-Scorer: A generalized model for predicting CRISPR on-target activity by machine learning and featurization. Journal of Molecular Cell Biology, 2019. (IF=4.7, in revision)

* Qiong Zhao, Jinbo Shen, Caiji Gao, Yong Cui, Yongyi Wang, Jie Cui, Lixin Cheng, Wenhan Cao, Ying Zhu, Shuxian Huang, Qianzi Zhou, Cheuk Ka Leong, King Pong Leung, Xuemei Chen & Liwen Jiang#. RST1 is a FREE1 Suppressor that Negatively Regulates Vacuolar Trafficking in Arabidopsis. Plant Cell, 2019. (IF=8.2)

* Lixin Cheng, Pengfei Liu, Dong Wang, and Kwong-Sak Leung#. Exploiting locational and topological overlap model to identify modules in human protein interaction network. BMC Bioinformatics, 2019. (IF=2.2)

* Lixin Cheng#, Kwong-Sak Leung#. Identification and characterization of moonlighting long non-coding RNAs based on RNA and protein interactome. Bioinformatics, 2018. (IF=7.3)

* Lixin Cheng#, Kwong-Sak Leung#. Quantification of non-coding RNA target localization diversity and its application in cancers. Journal of Molecular Cell Biology, 2017. (IF=6.0)

* Lixin Cheng, Pengfei Liu, and Kwong-Sak Leung#. SMILE: a novel procedure for subcellular module identification with localization expansion. IET System Biology, 2017 (Also in ACM-BCB 2017).

* Lixin Cheng, Kaili Fan, Yan Huang, Dong Wang#, and Kwong-Sak Leung#. Full characterization of localization diversity in human protein interactome. Journal of Proteome Research, 2017. (IF=4.3)

* Lixin Cheng, Xuan Wang, Pak-Kan Wong, Kwan-Yeung Lee, Le Li, Bin Xu, Dong Wang#, and Kwong-Sak Leung#.   ICN: A normalization method for gene expression data considering the over-expression of informative genes. Molecular BioSystems, 2016. (IF=2.8)

* Lixin Cheng, Leung-Yau Lo, Nelson Tang, Dong Wang#, and Kwong-Sak Leung#.   CrossNorm: a novel normalization strategy for microarray data in cancers. Scientific Reports, 2016. (IF=5.6)

* J PlaniLam, TC Chow, YH Fan, B Garcia-Bloj, L Cheng, E Ingley, W Hancock, S Fanayan, DY Jin, YQ Song. High expression of PTPN21 in B-cell non-Hodgkin’s gastric lymphoma, a positive mediator of STAT5 activity. Blood Cancer Journal, 2015. (IF=4.4)

* Y Li, C Wang, Z Miao, X Bi, D Wu, N Jin, L Wang, K Qian, C Li, T Zhang, C Zhang, Y Yi, H Lai, Y Hu, L Cheng, KS Leung, X Li, F Zhang, K Li, X Li and D Wang.   ViRBase: a resource for virus–host ncRNA-associated interactions. Nucleic Acids Research, 2014. (IF=9.1)

* Y Wang, L Chen, B Chen, X Li, J Kang, K Fan, Y Hu, J Xu, L Yi, J Yang, L Cheng, Y Li, C Wang, K Li, X Li, J Xu, and D Wang.   Mammalian ncRNA-disease repository: a global view of ncRNA-mediated disease network. Cell Death Disease, 2013. (IF=6.0)

* Dong Wang, Lixin Cheng, Yuannv Zhang, Ruihong Wu, Mingyue Wang, Yunyan Gu, Wenyuan Zhao, Pengfei Li, Bin Li, Yujing Zhang, Hongwei Wang, Huang Yan, Chenguang Wang, and Zheng Guo.   Extensive up-regulation of gene expression in cancer: the normalised use of microarray data. Molecular BioSystems, 2012. (IF=3.3)

* Dong Wang, Lixin Cheng, Mingyue Wang, Ruihong Wu, Pengfei Li, Bin Li, Yuannv Zhang, Yunyan Gu, Wenyuan Zhao, Chenguang Wang, and Zheng Guo.   Extensive increase of microarray signals in cancers calls for novel normalization assumptions. Computational Biology and Chemistry, 2012. (IF=1.8)

* Dong Wang, Yuan Zhang, Huang Yan, Pengfei Li, Ming Wang, Lixin Cheng, Wen Zhang, Yujing Zhang, Bin Li, Chenguang Wang, and Zheng Guo.   Comparison of different normalization assumptions for analyses of DNA methylation data from the cancer genome. Gene, 2012. (IF=2.3)

* Dong Wang, Yan Huang, Lixin Cheng，Pengfei Li，Yunyan Gu，Ruihong Wu，Mingyue Wang，Bin Li，Chenguang Wang.   Revealing extensive up - regulation of gene expression in pancreatic cancer based on microarray data. Chinese Journal of Bioinformatics, 2012.

* Dong Wang, Ruihong Wu, Lixin Cheng, Pengfei Li, Mingyue Wang, Bin Li, Chenguang Wang, and Zheng Guo.  Analyses of DNA methylation data based on different normalization assumptions. Chinese Journal of Biomedical Engineering, 2011.

* Xue Gong, Ruihong Wu, Hongwei Wang, Xinwu Guo, Dong Wang, Yunyan Gu, Yuannv Zhang, Lixin Cheng, Chenguang Wang, and Zheng Guo. Evaluating the consistency of differential expression of miRNAs detected in human cancers. Molecular Cancer Therapeutics, 2011. (IF=5.2)

* Xue Gong, Ruihong Wu, Yuannv Zhang, Wenyuan Zhao, Lixin Cheng, Yunyan Gu, Lin Zhang, Jing Wang, Jing Zhu, and Zheng Guo. Extracting consistent knowledge from highly inconsistent cancer gene data sources. BMC Bioinformatics, 2010. (IF=3.03)

### Conference:
* Lixin Cheng, Pengfei Liu, and Kwong-Sak Leung. SMILE: a novel procedure for subcellular module identification with localization expansion. ACM-BCB’17, August 20-23, 2017, Boston, MA, USA.

