## Lixin Cheng (程立新)

<img align="right" width="130" height="180" src="https://github.com/easonlcheng/easonlcheng.github.com/blob/master/bighead1.jpg?raw=true">

Address: Shenzhen People's Hospital, Shenzhen, China.

Email: easonlcheng@gmail.com

ORCID:  0000-0002-9427-383X 

[GoogleScholar](https://scholar.google.com/citations?user=LkWAc9YAAAAJ&hl=zh-CN&oi=ao)

[ResearchGate](https://www.researchgate.net/profile/Lixin-Cheng)

`RA recruitment:` I am looking for Postdoc and Research Assistants to work together computational biology and bioinformatics. Please email me if interested.

## Research Interests:
* Diagnostic and prognostic biomarker identification
* Machine learning algorithms
* Data integration and transfer learning
* Systems biololgy and network analysis
* Multiomics

## Education:
* Jan 2014 – Jan 2018, PhD (Computational Biology), Department of Computer Science and Engineering, The Chinese University of Hong Kong.
* 2008 – 2011, MPhil (Bioinformatics), College of Bioinformatics Science and Technology, Harbin Medical University.
* 2003 – 2008, BSc (Biotechnology), Department of Biotechnology, Harbin Medical University.

## Selected Publications: 
(**\*** corresponding author; **#** co-first author; **IF**, Impact Factor)

* Ning Zhang#, Fengxia Yang#, Pengfei Zhao#, Nana Jin, Haonan Wu, Tao Liu, Qingshan Geng\*, Xiaojun Yang\*, and **Lixin Cheng\***.   
MrGPS: a m6A-related gene pair signature to predict the prognosis and immunological impact of glioma patients.  
***Briefings in Bioinformatics, 2024. (IF=9.5)***
[Full Text](https://doi.org/10.1093/bib/bbad498)

*	Ying Li#, Jing Gao#, Xubin Zheng#, Guole Nie, Jican Qin, Haiping Wang, Tao He, Åsa Wheelock, Chuan-Xing Li\*, **Lixin Cheng\***, Xun Li\*.  
Diagnostic prediction of portal vein thrombosis in chronic cirrhosis patients using data-driven precision medicine model.  
***Briefings in Bioinformatics, 2024. (IF=9.5)***
[Full Text](https://doi.org/10.1093/bib/bbad478)

* Jun Wang, Marc Horlacher, **Lixin Cheng\***, Ole Winther\*.   
DeepLocRNA: An interpretable deep learning model for predicting RNA subcellular localisation with domain-specific transfer-learning.  
***Bioinformatics, 2024 (IF=6.9)***
[Full Text](https://doi.org/10.1093/bioinformatics/btae065)

* Jize Xie#, Xubin Zheng#, Jianlong Yan#, Qizhi Li, Nana Jin, Shuojia Wang, Pengfei Zhao, Shuai Li, Wanfu Ding\*, **Lixin Cheng\***, Qingshan Geng\*.   
Deep learning model to discriminate diverse infection types based on pairwise analysis of host gene expression.   
***iScience, 2024 (IF=5.8)***
[Full Text](https://www.cell.com/iscience/fulltext/S2589-0042(24)01130-1)

* Shuojia Wang, Yilin Chen, Lijiao Xiong, Nana Jin, Pengfei Zhao, Zhen Liang\*, **Lixin Cheng\***, Lin Kang\*.
Multimorbidity measures associated with cognitive function among community dwelling older Chinese adults.  
***Alzheimer's & Dementia, 2024 (IF=14, in press)***
   
* Yining Liang, Jiayue Zhang, Dehong Luo\*, **Lixin Cheng\***, Yijin Wang\*.  
Deregulation of immune response contributing to fulminant hepatitis in HEV infected pregnant women.  
***Journal of Medical Virology, 2024 (IF=12.7)***

* Qizhi Li#, Xubin Zheng#, Jize Xie#, Ran Wang, Mengyao Li, Man-Hon Wong, Kwong-Sak Leung, Shuai Li, Qingshan Geng\*, and **Lixin Cheng\***.   
bvnGPS: a generalizable diagnostic model for acute bacterial and viral infection using integrative host transcriptomics and pretrained neural networks.   
***Bioinformatics, 2023 (IF=6.9)***
[Full Text](https://doi.org/10.1093/bioinformatics/btad109)

* Ran Wang, Xubin Zheng, Fangda Song, Man Hon Wong, Kwong Sak Leung, and **Lixin Cheng\***.   
Deciphering associations between gut microbiota and clinical factors using microbial modules.   
***Bioinformatics, 2023 (IF=6.9)***
[Full Text](https://doi.org/10.1093/bioinformatics/btad213)

*	**Lixin Cheng#\***, Haonan Wu#, Xubin Zheng, Ning Zhang, Pengfei Zhao, Ran Wang, Qiong Wu, Tao Liu, Xiaojun Yang, and Qingshan Geng\*.   
GPGPS: a robust prognostic gene pair signature of glioma ensembling IDH mutation and 1p/19q co-deletion.   
***Bioinformatics, 2023 (IF=6.9)***
[Full Text](https://doi.org/10.1093/bioinformatics/btac850) 

*	Xiaojun Liu#, Chengyin Hong#, Yichun Jiang#, Wei Li, Youlian Chen, Yonghui Ma, Pengfei Zhao, Tiyuan Li, Huaisheng Chen\*, Xueyan Liu\*, and **Lixin Cheng\***.   
Co-expression module analysis reveals high expression homogeneity for both coding and non-coding genes in sepsis.  
***BMC Genomics, 2023 (IF=4.4)***
[Full Text](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-023-09460-9)

*	Jun Wang\*, Marc Horlacher, **Lixin Cheng**, Ole Winther.   
RNA trafficking and subcellular localization – a review of mechanisms, experimental and predictive methodologies.  
***Briefings in Bioinformatics, 2023. (IF=9.5)***
[Full Text](https://doi.org/10.1093/bib/bbad249)

*	Qiong Wu#, Xubin Zheng#, Kwong-Sak Leung, Man-Hon Wong, Stephen Kwok-Wing Tsui\*, and **Lixin Cheng\***.  
meGPS: a multi-omics signature for hepatocellular carcinoma detection integrating methylome and transcriptome data.  
***Bioinformatics, 2022 (IF=6.9)***
[Full Text](https://doi.org/10.1093/bioinformatics/btac379) 

*	Ran Wang, Xubin Zheng, Jun Wang, Shibiao Wan, Fangda Song, Man-Hon Wong, Kwong-Sak Leung, and **Lixin Cheng\***.  
Improving bulk RNA-seq classification by transferring gene signature from single cells in acute myeloid leukemia.   
***Briefings in Bioinformatics, 2022. (IF=14)***
[Full Text](https://doi.org/10.1093/bib/bbac002)

* Haili Li#, Xubin Zheng#, Jing Gao, Kwong-Sak Leung, Man-Hon Wong, Shu Yang, Yakun Liu, Ming Dong, Huimin Bai, Xiufeng Ye\*, and **Lixin Cheng\***.  
Whole transcriptome analysis reveals non-coding RNA’s competing endogenous gene pairs as novel form of motifs in serous ovarian cancer.   
***Computers in Biology and Medicine, 2022 (IF=6.9)***
[Full Text](https://doi.org/10.1016/j.compbiomed.2022.105881)

* Xueyan Liu, Yong Xu, Ran Wang, Sheng Liu, Jun Wang, YongLun Luo, Kwong-Sak Leung, and **Lixin Cheng\***.  
A network-based algorithm for the identification of moonlighting non-coding RNAs and its application in sepsis.  
***Briefings in Bioinformatics, 2021. (IF=11.6)***
[Full Text](https://doi.org/10.1093/bib/bbz154)

*	Yueran Yang#, Yu Zhang#, Shuai Li, Xubin Zheng, Man-Hon Wong, Kwong-Sak Leung, and **Lixin Cheng\***.  
A robust and generalizable immune-related signature for sepsis diagnostics.  
***IEEE/ACM Transactions in Computational Biology and Bioinformatics, 2021.*** 
[Full Text](https://pubmed.ncbi.nlm.nih.gov/34437068/)

*	Xubin Zheng, Kwong-Sak Leung, Man-Hon Wong, and **Lixin Cheng\***.  
Long non-coding RNA Pairs to Assist in Diagnosing Sepsis.  
***BMC Genomics, 2021.(IF=3.9)*** 
[Full Text](https://doi.org/10.1186/s12864-021-07576-4)

*	Honglong Wu#, Xuebin Wang#, Mengtian Chu, Dongfang Li, **Lixin Cheng\***, Ke Zhou\*.  
HCMB: A stable and efficient algorithm for processing the normalization of highly sparse Hi-C contact data.  
***Computational and Structural Biotechnology Journal, 2021. (IF=7.2)*** 
[Full Text](https://doi.org/10.1016/j.csbj.2021.04.064)

*	**Lixin Cheng#\***, Yonglun Zeng#, Shuai Hu, Ning Zhang, Kenneth C.P. Cheung, Baiying Li, Kwong-Sak Leung, Liwen Jiang\*.  
Systematic prediction of autophagy-related proteins using Arabidopsis thaliana interactome data.  
***Plant Journal 2021. (IF=6.4)*** 
[Full Text](https://doi.org/10.1111/tpj.15065)

* Yang Song, Suzhu Zhu, Ning Zhang, and **Lixin Cheng\***.  
Blood circulating miRNA pairs as a robust signature for early detection of esophageal cancer.  
***Front. Oncol., 2021. (IF=6.2)***
[Full Text](https://doi.org/10.3389/fonc.2021.723779)

*	Xubin Zheng#, Qiong Wu#, Haonan Wu#, Man-Hon Wong, Kwong Sak Leung, Xueyan Liu\*, and **Lixin Cheng\***.  
Evaluating the consistency of gene methylation in liver cancer using bisulfite sequencing data.  
***Front. Cell Dev. Biol., 2021. (IF=6.6)***
[Full Text](https://doi.org/10.3389/fcell.2021.671302)

*	Baiying Li#, Yonglun Zeng#, Wenhan Cao, Wenxin Zhang, **Lixin Cheng**, Haidi Yin, Qian Wu, Xiangfeng Wang, Yan Huang, Wilson Chun Yu Lau, Zhong-Ping Yao, Yusong Guo\*, Liwen Jiang\*.    
A distinct giant coat protein complex II vesicle population in Arabidopsis thaliana.  
***Nature Plants, 2021.***
[Full Text](https://www.nature.com/articles/s41477-021-00997-9)

*	Xueyan Liu#, Xubin Zheng#, Jun Wang#, Ning Zhang, Kwong-Sak Leung, Xiufeng Ye\*, and **Lixin Cheng\***.  
A long non-coding RNA signature for diagnostic prediction of sepsis upon ICU admission.  
***Clinical and Translational Medicine, 2020. (IF=7.9)***
[Full Text](https://doi.org/10.1002/ctm2.123)

*	Sheng Liu, Wenjing Zhao, Xueyan Liu\*, and **Lixin Cheng\***.  
Metagenomic analysis of the gut microbiome in atherosclerosis patients identify cross-cohort microbial signatures and potential therapeutic target.  
***FASEB Journal, 2020. (IF=5.0)***
[Full Text](https://doi.org/10.1096/fj.202000622R)

*	Linting Li, Min Liu, Ludan Yue, Rui Wang, Ning Zhang, Yujie Liang, Liang Zhang, **Lixin Cheng\***, Jiang Xia\*, Ruibing Wang\*.  
Host–Guest Protein Assembly for Affinity Purification of Methyllysine Proteomes.  
***Analytical Chemistry, 2020. (IF=6.8)***
[Full Text](https://doi.org/10.1021/acs.analchem.0c01643)

*	Miao Liu, Sicong He, **Lixin Cheng**, Jianan Qu, and Jiang Xia\*.  
Phase-Separated Multienzyme Biosynthesis.  
***Biomacromolecules, 2020 (IF=6.1)***

* **Lixin Cheng#**, Chuanchuan Nan#, Lin Kang#, Ning Zhang, Sheng Liu, Huaisheng Chen, Chengying Hong, Youlian Chen, Zhen Liang\*, Xueyan Liu\*.  
Whole blood transcriptomic investigation identifies long non-coding RNAs as regulators in sepsis.  
***Journal of Translational Medicine, 2020. (IF=4.1)***
[Full Text](https://doi.org/10.1186/s12967-020-02372-2)

*	Kenneth Cheung, Silvia Fanti, Claudio Mauro, GS Wang, Anitha Nair, HM Fu, Silvia Angeletti, Silvia Spoto, Marta Fogolari, Francesco romano, Dunja Aksentijevic, WW Liu, BY Li, **Lixin Cheng**, LW Yang, Juho Vuononvirta, Thanushiyan Poobalasingam, David Smith, Massimo Ciccozzi, Egle Solito, Federica Marelli-Berg.  
Preservation of microvascular barrier function requires CD31 receptor-induced metabolic reprogramming.  
***Nat Commun. 2020 Jul 17;11(1):3595.***

* Jun Wang, Xiuqing Zhang, **Lixin Cheng\***, Yonglun Luo\*.  
An overview and metanalysis of Machine and Deep Learning-Based CRISPR gRNADesign Tools.  
***RNA Biology, 2019. (IF=5.5)***
[Full Text](https://doi.org/10.1080/15476286.2019.1669406)

* Jun Wang, Xi Xiang, Lars Bolund, Xiuqing Zhang, **Lixin Cheng\***, Yonglun Luo\*.  
GNL-Scorer: A generalized model for predicting CRISPR on-target activity by machine learning and featurization.  
***Journal of Molecular Cell Biology, 2019. (IF=4.7)***
[Full Text](https://doi.org/10.1093/jmcb/mjz116)

* Xueyan Liu#, Nan Li#, Sheng Liu, Jun Wang, Ning Zhang, Xubin Zheng, Kwong-Sak Leung, and **Lixin Cheng\***.  
Normalization methods for the analysis of unbalanced transcriptome data: a review.  
***Frontier in Biotechnology and Bioengineering, 2019. (IF=5.1)***
[Full Text](https://doi.org/10.3389/fbioe.2019.00358)

* Qiong Zhao, Jinbo Shen, Caiji Gao, Yong Cui, Yongyi Wang, Jie Cui, **Lixin Cheng**, Wenhan Cao, Ying Zhu, Shuxian Huang, Qianzi Zhou, Cheuk Ka Leong, King Pong Leung, Xuemei Chen & Liwen Jiang\*.  
RST1 is a FREE1 Suppressor that Negatively Regulates Vacuolar Trafficking in Arabidopsis.  
***Plant Cell, 2019.***

* **Lixin Cheng**, Pengfei Liu, Dong Wang\*, and Kwong-Sak Leung\*.  
Exploiting locational and topological overlap model to identify modules in human protein interaction network.  
***BMC Bioinformatics, 2019. (IF=2.2)***
[Full Text](https://doi.org/10.1186/s12859-019-2598-7)

* **Lixin Cheng\***, Kwong-Sak Leung\*.  
Identification and characterization of moonlighting long non-coding RNAs based on RNA and protein interactome.  
***Bioinformatics, 2018. (IF=7.3)***
[Full Text](https://doi.org/10.1093/bioinformatics/bty399)

* **Lixin Cheng\***, Kwong-Sak Leung\*.  
Quantification of non-coding RNA target localization diversity and its application in cancers.  
***Journal of Molecular Cell Biology, 2017. (IF=6.0)***
[Full Text](https://doi.org/10.1093/jmcb/mjy006)

* **Lixin Cheng**, Pengfei Liu, and Kwong-Sak Leung\*.  
SMILE: a novel procedure for subcellular module identification with localization expansion.  
***IET System Biology, 2017 (Also in ACM-BCB 2017)***
[Full Text](https://doi.org/10.1049/iet-syb.2017.0085)

* **Lixin Cheng**, Kaili Fan, Yan Huang, Dong Wang\*, and Kwong-Sak Leung\*.  
Full characterization of localization diversity in human protein interactome.  
***Journal of Proteome Research, 2017. (IF=4.3)***
[Full Text](https://doi.org/10.1021/acs.jproteome.7b00306)

* **Lixin Cheng#**, Xuan Wang#, Pak-Kan Wong, Kwan-Yeung Lee, Le Li, Bin Xu, Dong Wang\*, and Kwong-Sak Leung\*.  
ICN: A normalization method for gene expression data considering the over-expression of informative genes.  
***Molecular BioSystems, 2016. (IF=2.8)***
[Full Text](https://doi.org/10.1039/C6MB00386A)

* **Lixin Cheng**, Leung-Yau Lo, Nelson Tang\*, Dong Wang\*, and Kwong-Sak Leung\*.  
CrossNorm: a novel normalization strategy for microarray data in cancers.  
***Scientific Reports, 2016. (IF=5.6)***
[Full Text](https://doi.org/10.1038/srep18898)

### Conference:
* **Lixin Cheng**, Pengfei Liu, and Kwong-Sak Leung.  
SMILE: a novel procedure for subcellular module identification with localization expansion.  
***ACM-BCB’17, August 20-23, 2017, Boston, MA, USA.***


